examples/platform/silabs/efr32/BUILD.gn

```diff
@@ -284,7 +287,16 @@ source_set("efr32-common") {
   }
 
   if (chip_enable_wifi) {
-    if (use_rs911x) {
+    if (use_rs9116) {
```

Comment:
why separate in two if/else if if both statement does the same ?

Reply:
Modifed Now